src/GeositeFramework/plugins/layer_selector/layers.json

```diff
@@ -1,6 +1,51 @@
 [
     {
         "agsSource": {
+            "availableInRegions": ["galveston"],
+            "url": "http://services.coastalresilience.org:6080/arcgis/rest/services",
+            "folderTitle": "Galvestone Local",
+            "folders": [
+                {
+                    "url": "http://services.coastalresilience.org:6080/arcgis/rest/services/Global",
+                    "name": "",  // Don't show a folder for these services
```

Comment:
Yeah, I was surprised to see the comments in there.  These are just copied from the main layer json, which appear to work as far as the parsing and the validator is concerned.